blockchain-source/src/lib.rs

```diff
@@ -1,4 +1,11 @@
 pub mod cardano;
+pub mod multiverse;
 mod source;
 
 pub use source::*;
+
+pub trait GetNextFrom {
```

Comment:
I wonder whether this is self-explanatory enough

Reply:
we can rename to sth like GetNextSourceFrom

Reply:
I think that's a bit mouthful and redundant... It's already in the source module after all. Mostly I was wondering if a doc comment was necessary, but I think it's fine this way.